### PR TITLE
Short socket names to prevent 104 char limit

### PR DESCRIPTION
**Pull request by ep1kt3t0s**
Bug fix that prevents FFMpegCore to run on MacOS.
Discussed at https://github.com/rosenbjerg/FFMpegCore/issues/365

### CODE REVIEW

**Comment by ep1kt3t0s** [comment]
Is this project dead, @rosenbjerg ?

File: FFMpegCore/FFMpeg/Pipes/PipeHelpers.cs [hunk 1]
```diff
@@ -1,18 +1,20 @@
 ﻿using System;
+using System.IO;
 using System.Runtime.InteropServices;
 
 namespace FFMpegCore.Pipes
 {
     static class PipeHelpers
     {
-        public static string GetUnqiuePipeName() => $"FFMpegCore_{Guid.NewGuid()}";
+        public static string GetUnqiuePipeName() => $"FFMpegCore_{Guid.NewGuid().ToString("N").Substring(0, 5)}";
```
Comment (rosenbjerg) [inline FFMpegCore/FFMpeg/Pipes/PipeHelpers.cs:9]: How is uniqueness guarenteed with only 5 chars?

Comment (ep1kt3t0s) [reply FFMpegCore/FFMpeg/Pipes/PipeHelpers.cs:9]: Uniqueness is not guaranteed even with 32 or 36+ characters as well.
Considering this is a media processing library, the temporary files should be disposed of after each run and I don't see how a single disk can be used for highly concurrent media processing that can overlap with 5-character randomness.
Even if that happens, any implementer of a media processing solution should always have a fail-safe and retry logic, regardless of this.
So, I see generating longer names that breaks this library running on Mac does not provide any value.
But if you truly want to provide uniqueness, you can re-call this function to re-generate another name if the path already exists. Even in that case, you need a good locking mechanism.

Comment (rosenbjerg) [reply FFMpegCore/FFMpeg/Pipes/PipeHelpers.cs:9]: True, but the chance of it being unique is higher the more characters. Anyway, I agree that it probably won't cause any problems, and as the current solution does cause problems on macos I'm fine with this change

**Comment by rosenbjerg** [comment]
Merged, thanks

**Pull request merged**